src/components/date-picker-month/date-picker-month.tsx

```diff
@@ -277,10 +279,16 @@ export class DatePickerMonth {
     const date = lastDate.getDate();
```

Comment:
Unrelated nitpick: this method is not following our casing convention and should be `getPrevMonthDays` (not `getPrevMonthdays`).